src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -50,3 +50,12 @@ types.h = {a: 'a', b: 34};
  * @template T
  */
 types.identity = function(a) { return a; };
+
+/** @type {Object<number, string>} */
+types.j = {a: 'a'};
+
+/**
+ * @param {Object<K,V>} obj The object in which to look for val.
+ * @template K,V
+ */
+types.containsValue = function(obj) { };
```

Comment:
Nit: in general, I think it'd be nicer to have tests that express some concept, instead of just being C&P from the Closure library. It'll make it easier to understand this tests in the future. E.g. here: `types.objectWithGenericKeyType` or so?